File: Submissions/#0375_div2/A/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    int x1 = sc.nextInt();
    int x2 = sc.nextInt();
    int x3 = sc.nextInt();
    
    int max = Math.max(x1, x2);
    int min = Math.min(x1, x2);

    max = Math.max(max, x3);
    min = Math.min(min, x3);

    System.out.println(max - min);

  }

}

File: Submissions/#0375_div2/A/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  int x1, x2, x3;
  cin >> x1 >> x2 >> x3;

  int ma = max(x1, x2);
  int mi = min(x1, x2);

  ma = max(ma, x3);
  mi = min(mi, x3);
  
  cout << ma - mi << endl;

}